docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/upgradability-guidelines.md

```diff
@@ -29,23 +31,23 @@ In the example, the name is `PrivateApi:Extension`. To find the documentation fo
+| PrivateApi |                                                                                                                                     | PrivateApi:Facade                      | Please avoid usage of **{method_name}**(...) in **{class_name}**                                                                                                   | [Private API is extended](/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.html)                                                  |
+| PrivateApi |                                                                                                                                     | PrivateApi:MethodIsOverwritten         | Please avoid usage of core method **{class_namespace}::{method_name}** in the class **{class_namespace}**                                                          | [Private API method is overridden on the project level](/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.html)                    |
+| PrivateApi |                                                                                                                                     | PrivateApi:ObjectInitialization        | Please avoid usage of **{class_namespace}** in **{class_namespace}**                                                                                               | [Private API is extended](/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.html)                                                  |
+| PrivateApi |                                                                                                                                     | PrivateApi:Persistence                 | Please avoid usage of $this->**{method_name}**(...) in **{class_name}**                                                                                            | [Private API is used on the project level](/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.html#privateapi:persistence)          |
+| PrivateApi |                                                                                                                                     | PrivateApi:PersistenceInBusinessModel  | Please avoid usage of PrivateApi **{object_name}**->**{method_name}(...)** in **{class_name}**                                                                     | [Private API is extended](/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.html)                                                  |
```

Comment:
If you use here everywhere for `PrivateApi` the links with anchors, that go directly to the certain rule, and not just on the page, that improves user experience and will be great

Reply:
Yes, I will do it 
Just waiting answer from Andrii, on how to correctly put them (for getting green sniffer)

Reply:
Added anchors